search_module.py

```diff
@@ -5,14 +5,22 @@
 
 INDEX_NAME = 'institution'
 
+"""This method receives an id, which is the key.urlsafe() of the institution,
```

Comment:
def complex(real=0.0, imag=0.0):
    """Form a complex number.
    Keyword arguments:
    real -- the real part (default 0.0)
    imag -- the imaginary part (default 0.0)
    """
    if imag == 0.0 and real == 0.0:
        return complex_zero

Exemplo de documentação com parametros, faz nesse estilo :)